liferay-echo/liferay-portal
# LPS-196022 Adds a page with collection fragments

## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-196022)

The goal of the task is to add a page with collections to DataFactory so that performance tests can be executed on it

## Change summary:

* Modify DataFactory class to add a page with collections. We create models for AssetListEntry and related entities to achieve this


## Test Scenario

* Fetch the pull request
* Go to benchmark folder in your source code for liferay portal
* To make it easier and faster to test, you can open benchmark.properties and change sample.sql.max.commerce.group.count to 0 and sample.sql.max.group.count to 1
* Execute ant build-sample-sql
* After this finishes, import sample-mysql.sql into your database
* Start the portal
* Go to Search and reindex BlogsEntry (In Apps Menu > Control Panel > Search)
* Go to Apps Menu > Control Panel > Sites, and go to the Pages of Site 1
* In the navigation menu, click on collection_page
* You will see 21 collections displaying the blogs of the site
